ePBFT/src/pbft/bft-simple/test/ledger_reader_main.cpp

```diff
@@ -3,35 +3,16 @@
 #include "Digest.h"
```

Comment:
This does not look like a unit test, am I missing something?

Reply:
It is not a unit test, this was used originally to read from the ledger and it is used in `e2e_test.py` right now to make some checks. It will go away soon and I am happy to delete it now if @ashamis doesn't find a need for it, although we might need to add some code for the e2e tests to pass.

Reply:
Ok, let's delete it now then. As agreed with @sylvanc , we are not making any additions or changes to tests other than unit tests that are PBFT-specific.

Reply:
As discussed with @ashamis this will stay here a bit longer as it is still used in an e2e tests and is useful for debugging, but will go away soon

Reply:
Removed much talked about `leger_reader_main.cpp` file along with the e2e test that uses it

Reply:
Thank you for addressing this.